lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -180,6 +180,23 @@ describe('minification estimator', () => {
       assert.equal(computeJSTokenLength(js), 9);
     });
 
+    it('should handle regular expression character classes', () => {
+      const js = `
+        /regex [^/]\\//.test('this should be in string not comment 123456789')
+      `;
+
+      assert.equal(computeJSTokenLength(js), 69);
+      assert.equal(computeJSTokenLength(js), js.trim().length);
+    });
+
+    it('should handle escaped regular expression characters', () => {
+      const js = `
+        /regex \\[/ // this should be in comment not string 123456789
```

Comment:
I'm also confused here?  What is this testing?  The comment is still part of the regex
```javascript
/regex \\[/ // this should be in comment not string 123456789
```
If it was supposed to be a string then it needs to escape the `[`
```javascript
/regex \\\[/ // this should be in comment not string 123456789
```

Reply:
same deal here it's the js equivalent of
```js
/regex \[/ // this should be in comment not string 123456789